plotnine/stats/stat_qq.py

```diff
@@ -78,6 +78,6 @@ def compute_group(cls, data, scales, **params):
 
         quantiles = np.asarray(quantiles)
         cdist = get_continuous_distribution(params['distribution'])
-        theoretical = cdist.ppf(quantiles, *params['dparams'])
+        theoretical = cdist.ppf(quantiles, **params['dparams'])
```

Comment:
You need to change the [default](https://github.com/has2k1/plotnine/pull/636/files#diff-5374bfbe62ced2d754f62c16a97624db7b812869ab590bf1a230b521046a7162R63) value for `dparams` to be a `{}`.

Reply:
thanks you, I've changed it